packages/components/src/tooltip/index.js

```diff
@@ -12,6 +12,7 @@ import {
 	cloneElement,
 	concatChildren,
 } from '@wordpress/element';
+import { KeyboardShortcuts } from '@wordpress/components';
```

Comment:
@aduth, this is a little bit strange, just by importing `KeyboardShortcuts` I do get that nasty error:
`Fatal error: Allowed memory size of 268435456 bytes exhausted ...`
Am I doing something wrong here?
![Peek 2019-05-20 16-57](https://user-images.githubusercontent.com/1477453/58027227-db4df500-7b20-11e9-838a-bd9679fadaa1.gif)

Reply:
you can't import something from the same package using the `@wordpress/*` syntax, just use relative paths.

Reply:
Discussed in Slack thread ([link requires registration](https://make.wordpress.org/chat/)):

https://wordpress.slack.com/archives/C02QB2JS7/p1557863367231300